netty/src/main/java/io/grpc/transport/netty/Utils.java

```diff
@@ -177,13 +173,8 @@ private static Http2Headers convertMetadata(Metadata headers) {
 
     @Override
     public EventLoopGroup create() {
-      return new NioEventLoopGroup(0, new ExecutorServiceFactory() {
```

Comment:
Oh, I see.

Using ThreadFactory constructor seems more what we want:
```java
return new NioEventLoopGroup(0,
    new ThreadFactoryBuilder().setNameFormat(name + "-%d").build());
```

That works, right?